oval_graph/command_line_client/client_html_output.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, args):
         self.display_html = True if self.out is None else self.arg.display
         self.html_builder = Graph(self.arg.verbose, self.all_in_one)
         self.web_browsers = []
+        self.is_chosen_more_rules = False
```

Comment:
probably `selected_multiple_rules` would be a better name, but because you use it in a negation, why not to have `selected_only_one_rule` instead?